tests/k8s_env/configmap/kubearmor_config_test.go

```diff
@@ -273,53 +273,53 @@ var _ = Describe("KubeArmor-Config", func() {
 
 		})
 
-		It("default posture will be unchanged after global configs changed", func() {
-
-			// apply a allow based policy
-			err := K8sApplyFile("manifests/ksp-fullyAnnotated-allow.yaml")
-			Expect(err).To(BeNil())
-
-			err = KarmorLogStart("policy", "fullyannotated", "Network", fullyAnnotated)
-			Expect(err).To(BeNil())
-
-			// initialy namespace defaults posture is block (annotated fully)
-			sout, _, err := K8sExecInPodWithContainer(fullyAnnotated, "fullyannotated", "ubuntu-1", []string{"bash", "-c", "curl google.com"})
-			Expect(err).To(BeNil())
-			fmt.Printf("---START---\n%s---END---\n", sout)
-			Expect(sout).To(MatchRegexp(".*has moved"))
-
-			// should get an alert with success
-			// check policy violation alert
-
-			target := protobuf.Alert{
-				PolicyName:    "DefaultPosture",
-				Action:        "Audit",
-				Result:        "Passed",
-				NamespaceName: "fullyannotated",
-			}
-
-			res, err := KarmorGetTargetAlert(5*time.Second, &target)
-			Expect(err).To(BeNil())
-			Expect(res.Found).To(BeTrue())
-
-			// change global default posture to block using configmap
-			cm := NewDefaultConfigMapData()
-			cm.DefaultFilePosture = "block"
-			cm.DefaultCapabilitiesPosture = "block"
-			cm.DefaultNetworkPosture = "block"
-			err = cm.CreateKAConfigMap() // will create a configMap with default posture as block
-			Expect(err).To(BeNil())
-
-			// wait for policy updation due to defaultPosture change
-			time.Sleep(5 * time.Second)
-
-			// defaults posture should still be audit for network
-			sout, _, err = K8sExecInPodWithContainer(fullyAnnotated, "fullyannotated", "ubuntu-1", []string{"bash", "-c", "curl google.com"})
-			Expect(err).To(BeNil())
-			fmt.Printf("---START---\n%s---END---\n", sout)
-			Expect(sout).To(MatchRegexp(".*has moved"))
-
-		})
+		// It("default posture will be unchanged after global configs changed", func() {
```

Comment:
so we're keeping commented tests those are not moved to networktest right!

Reply:
missed it, I'll remove them.

Reply:
removed them completely as it would be redundant to setup config tests in networktests.